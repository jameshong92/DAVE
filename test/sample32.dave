// test fld type conversion
int main() {
	str path = "test.txt";
    tbl test = load(path);
    test = convert(test, 0, "string");
    print(test);
    test = convert(test, 2, "int");
    print(test);
}