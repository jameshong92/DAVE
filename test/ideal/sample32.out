height	name	gender	weight	;
178	tom	male	68	;
195	jacky	male	81	;
175	jay	male	77	;
60	janice	female	48	;
174	sammy	female	69	;

height	name	gender	weight	;
178	tom	male	68	;
195	jacky	male	81	;
175	jay	male	77	;
60	janice	female	48	;
174	sammy	female	69	;

